tests/pipelines/stepping.py

```diff
@@ -106,7 +106,7 @@ def _enqueue(self, name, item):
         for item in items:
             if type(item) != tuple:
                 item = (item,)
-            queue.append(item)
+            queue.append(list(item))
```

Comment:
It goes tuple -> list

Alternately, it would be written as:

```python
if type(item) == tuple:
    item = list(item)
else:
    item = [item]
```